cdc-debezium/README.md

```diff
@@ -45,17 +45,24 @@ kind: Spicepod
 name: cdc-debezium
 
 datasets:
-  - from: debezium:cdc.public.customer_addresses
+  - from: debezium:cdc.inventory.customer_addresses
     name: cdc
     params:
       debezium_transport: kafka
       debezium_message_format: json
       kafka_bootstrap_servers: localhost:19092
+      kafka_security_protocol: SASL_SSL
+      kafka_sasl_mechanism: SCRAM-SHA-512
+      kafka_sasl_username: kafka
+      kafka_sasl_password: kafka_password
+      kafka_ssl_ca_location: ./certs/kafka_ca_cert.pem
+      kafka_enable_ssl_certificate_verification: true
```

Comment:
I think you accidentally copied the SASL spicepod here?

Reply:
ah, right